behave/runner.py

```diff
@@ -598,6 +649,69 @@ def stop_capture(self):
     def teardown_capture(self):
         self.capture_controller.teardown_capture()
 
+    def run_one_feature(self, feature):
+        try:
+            self.feature = feature
+            for formatter in self.formatters:
+                formatter.uri(feature.filename)
+
+            def read_timeout(tags):
+                tag_name = "timeout="
+                for tag in tags:
+                    res = tag.find(tag_name)
+                    if res == 0:
+                        return int(tag[len(tag_name):])
+                return 0
+
+            timeout = read_timeout(feature.tags)
+            r, w = os.pipe()
+            newpid = os.fork()
```

Comment:
Indeed, this isn't portable. Make sure the implementation complies with the limitations of each platform https://docs.python.org/2/library/multiprocessing.html#windows

Reply:
Reviewing, thanks.